src/styles/global.css

```
@@ -1,62 +1,86 @@
+    --gap: var(--size-fluid-2);
+    --borderColor: oklch(from var(--text) l c h / 0.2);
+    --border: var(--border-size-1) solid var(--borderColor);
+    --roundness: 0.3rem;
+    --blur: blur(var(--size-px-3));
+    --player-bg: none;
+    --player-bp: 0 0;
+    --panelRatioLeft: 2;
+    --panelRatioRight: 5;
+    --trueBg: var(--bg);
+    --font:
+        ui-rounded, Segoe UI, source-sans-pro, Inter, Noto Sans, Oxygen,
```

Stylelint (17.14.0)

[error] Expected "Segoe" to be "segoe" (value-keyword-case)

(value-keyword-case)